specs/commandline/policy.md

```diff
@@ -0,0 +1,232 @@
+            ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=acme-rockets.io, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  import    import trust policies from a JSON file
+  init      initialize trust policies with default values
+  show      show trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy import
+
+```text
+Import trust policies from a JSON file
+
+Usage:
+  notation policy import [flags] <file_path>
+
+Flags:
+  -d, --debug     debug mode
+      --force     override the existing trust policies, never prompt
+  -h, --help      help for import
+  -v, --verbose   verbose mode
+```
+
+### notation policy init
+
+```text
+Initialize trust policies with default values
+
+Usage:
+  notation policy init [flags]
+
+Flags:
+  -d, --debug                      debug mode
+      --force                      restore the trust policies to default values. Any existing trust polices will be discarded, never prompt
+  -h, --help                       help for export
+      --trust-store stringArray    specify the trust stores in format "<type>:<name>", e.g. "ca:my_store". If this flag is ignored, a default trust store "ca:default" is used
+  -v, --verbose                    verbose mode
+```
+
+### notation policy show
+
+```text
+Show trust policies
+
+Usage:
+  notation policy show [flags]
+
+Flags:
+  -d, --debug     debug mode
+  -h, --help      help for show
+  -v, --verbose   verbose mode
+```
+
+## Usage
+
+### Initialize trust policies with default values
+
+```shell
+notation policy init
+```
+
+Upon successful execution, trust policies with default values are created and printed out as following. Use command `notation cert add --type ca --store default <cert_file>` to add CA certificates to trust store `ca:default`. If users want to use different trust stores, refer to section [Update trust policies](#update-trust-policies) on how to do the update.
+
+```jsonc
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "policy-by-init-command",
+            "registryScopes": ["*"],
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": ["ca:default"],
+            "trustedIdentities": ["*"]
+        }
+    ]
+}
+
+```
+
+If there are existing trust policies configured and users still run `notation policy init` command, A prompt should be displayed asking for confirmation on whether restoring to default values or not. Use `--force` flag to discard any existing trust policies without prompt.
+
+### Initialize trust policies with specified trust stores
+
+```shell
+notation policy init --ts "ca:my_store" --ts "ca:my_store_2"
```

Comment:
`--ts` is not specified or supported. Full flag name `--trust-store` should be used.